api/v1alpha1/driver_types.go

```diff
@@ -185,10 +185,10 @@ type DriverSpec struct {
 	Encryption *EncryptionSpec `json:"encryption,omitempty"`
 
 	// Driver's plugin configuration
-	Plugin *PluginSpec `json:"plugin,omitempty"`
+	NodePlugin *NodePlugin `json:"nodePlugin,omitempty"`
 
-	// Driver's provisioner configuration
-	Provisioner *ProvisionerSpec `json:"provisioner,omitempty"`
+	// Driver's controller plugin configuration
+	ControllerPlugin *ControllerPluginSpc `json:"controllerPlugin,omitempty"`
```

Comment:
we also need to change provisionerReplicas to ControllerPluginReplicas as well.

Reply:
It is already inside the ControllerPluginSpec section so we should just call it Replica